src/main/java/org/killbill/billing/plugin/adyen/api/mapping/KlarnaPaymentMappingService.java

```diff
@@ -65,6 +77,46 @@ public static PaymentInfo toPaymentInfo(final String merchantAccount, final Stri
         return lineItems;
     }
 
+    private static List<LineItem> getLineAdjustments(final List<LineItem> lineItems) {
+        final Map<String, LineAdjustment> adjustmentMap = new HashMap<>();
+        final List<LineItem> adjustmentLines = new ArrayList<>();
+        if(lineItems!= null && lineItems.size() > 0) {
```

Comment:
check using isEmpty()

Reply:
do we need this if

Reply:
Indentation `lineItems!= null` space is missing

Reply:
indentation updated. lineItem will be null for auth complete case.